distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/JvmOptionsParser.java

```diff
@@ -70,25 +71,26 @@ SortedMap<Integer, String> invalidLines() {
      * variable.
      *
      * @param args            the start-up arguments
-     * @param configDir       the ES config dir
+     * @param processInfo     info to pass through to the new Java subprocess
```

Comment:
The processInfo is about the current process, ie the cli, not the Java process we will create.

Reply:
I copied-pasted it from another description, thanks for correcting this